src/stylesheets/components/_list-styles.scss

```diff
@@ -249,6 +249,24 @@ dl, dt, dd {
   }
 }
 
+// List style for <ul> tags without any classes, nested
+// inside elements with class `.list--{{modifier}}` or
+// <ol> tags.
+
+[class*=list--], ol {
+  ul, ul[class=""] {
```

Comment:
nested style does not work as describe in comments, see https://parliament-uk-pugin.herokuapp.com/templates/prototypes/_search-results.html